README.md

```diff
@@ -307,3 +312,87 @@ Class | Method | HTTP request | Description
 
 - **Type**: HTTP basic authentication
 
+
+<a name="documentation-for-serialization-deserialization"></a>
+## Documentation for Serialization/Deserialization for Enums
+
+###These methods are not required unless being used for debugging/logging purposes
+
+- To serialize (turn value into enum), we would need to use the reflection method GetEnumByValue where you pass the enum as a type and value into the method to get the associated enum.
+  
+  GetMemberValue
+  
+  This is an extension method, so you can turn an enum directly into a value
+
+  Get a value from an enum
+
+  ```csharp
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System;
+    using System.Linq;
+    using System.IO;
+    using freeclimb.Api;
+    using freeclimb.Client;
+    using freeclimb.Model;
+    using System.Reflection;
+    using Newtonsoft.Json;
+
+    using freeclimb.Enums;
+
+    namespace Example
+    {
+    public class GetMemberValueExample
+    {
+        public static void Main()
+        {
+            AccountStatus status = AccountStatus.ACTIVE;
+            string expectedValue = StringValueAccountStatusTestValue();
+            string calculatedValue = status.GetMemberValue();
+            Assert.Equal(expectedValue, calculatedValue);
+        }
+        private string StringValueAccountStatusTestValue()
+        {
+            return "active";
+        }
+    }
+
+    }
+    ```
+    
+- To deserialize a value (turn enum into value), we would need to use the extension method GetMemberValue where you pass the enum as a parameter into the method to get the associated value.
+
+  GetEnumByValue
+  
+  Reflection.GetEnumByValue<EnumType>(expectedValue)
+
+  Get an Enum value from string
+
+  ```csharp
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System;
+    using System.Linq;
+    using System.IO;
+    using freeclimb.Api;
+    using freeclimb.Client;
+    using freeclimb.Model;
+    using System.Reflection;
+    using Newtonsoft.Json;
+
+    using freeclimb.Enums;
+
+    namespace Example
+    {
+    public class GetEnumByValueExample
+    {
+        public static void Main()
+        {
+            string expectedValue = "active";
+            AccountStatus calculatedValue = Reflection.GetEnumByValue<AccountStatus>(expectedValue);
+            Assert.Equal(instance, calculatedValue);
```

Comment:
this example doesn't make sense. 
You're creating a `calculatedValue` enum - the assert statement would fail because an enum is not going to equal a string

Reply:
The instance is being compared to the calcuatedValue, enum to enum, not enum to string. Maybe a rename of the variables will help here.